plugins/org.jboss.tools.openshift.ui/src/org/jboss/tools/openshift/internal/ui/wizard/connection/OAuthDetailView.java

```diff
@@ -212,6 +212,11 @@ public String toString() {
 		}
 
 		@Override
+		public void widgetDefaultSelected(SelectionEvent e) {
+			widgetDefaultSelected(e);
+		}
```

Comment:
Did you mean to call widgetDefaultSelected(e) from inside widgetDefaultSelected(e)? 

Or was this supposed to call widgetSelected instead?